Elsewhen/Elements/MykeMode/SelectedTimeZoneCell.swift

```diff
@@ -10,37 +10,21 @@ import UniformTypeIdentifiers
 
 struct SelectedTimeZoneCell: View {
     let tz: TimeZone
-    let flag: String
-    let timeInZone: String
-    let selectedDate: Date
+    
+    let formattedString: String
     
     var body: some View {
-        let abbreviation = tz.fudgedAbbreviation(for: selectedDate)
         VStack(alignment: .leading) {
-            HStack {
-                VStack(alignment: .leading) {
-                    Text(tz.friendlyName)
-                        .font(.caption)
-                        .foregroundColor(.secondary)
-                    Text("\(flag) \(timeInZone)")
-                        .onDrag {
-                            return NSItemProvider(object: timeInZone as NSString)
-                        }
-                }
-                Spacer()
-                if let abbreviation = abbreviation {
-                    Text(abbreviation)
-                        .foregroundColor(.secondary)
-                }
-            }
+            Text(tz.friendlyName)
+                .font(.caption)
+                .foregroundColor(.secondary)
+            Text(formattedString)
             .onDrag {
                 let tzItemProvider = tz.itemProvider
                 let itemProvider = NSItemProvider(object: tzItemProvider)
                 itemProvider.suggestedName = tzItemProvider.resolvedName
                 return itemProvider
             }
         }
-        .accessibilityElement()
-        .accessibilityLabel(Text("\(tz.friendlyName): \(timeInZone), \(abbreviation ?? "")"))
```

Comment:
Have we checked the accessibility tags are still okay?

Reply:
I meant to put these back afterwards 🤦